### PR TITLE
Working on docs

### PR DESCRIPTION
**Pull request by oarrabi**
Working on readme

@goyox86

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -363,6 +370,17 @@ Notice how the command the sub-command and flag info are displayed.
 
 Read more about the [help message](http://docs.getguaka.com/Protocols/HelpGenerator.html)
 
+## Cross-Platform utility libraries aka batteries
+
+Writing a command line application is more than just parsing the command line arguments and flags. 
+
+Swift ecosystem is still very young e.g no cross platform stdlib and we did not wanted to make Guaka depend on libFoundation, so we rolled up our sleeves and made these babies cross-platform swift libraries so you can be productive instantaneously:
```
Comment (goyox86) [inline Readme.md]: "a few small cross-platform Swift libraries" instead of "these babies cross-platform swift libraries"

Comment (goyox86) [inline Readme.md]: Wat da ya thank? Brasnar?

Comment (goyox86) [inline Readme.md]: Actually this: 

"Swift ecosystem is still very young and lacks of a cross-platform standard library. We did not wanted to make Guaka depend on libFoundation, so we rolled up our sleeves and built a few small cross-platform (as in whenever there is a usable C standard library) libraries. so you don't have to and can be productive instantaneously. Also , they are usable on their own. You are welcome to use them too! <3. "

Comment (oarrabi) [inline Readme.md]: DONE!! much <3

**Pull request merged**